velox/common/memory/SharedArbitrator.cpp

```diff
@@ -200,6 +197,16 @@ void SharedArbitrator::updateFreeCapacityMetrics() const {
       kMetricArbitratorFreeReservedCapacityBytes, freeReservedCapacity_);
 }
 
+void SharedArbitrator::updateArbitrationRequestStats() {
+  ++numRequests_;
+  RECORD_METRIC_VALUE(kMetricArbitratorRequestsCount);
+}
+
+void SharedArbitrator::updateArbitrationFailureStats() {
+  RECORD_METRIC_VALUE(kMetricArbitratorFailuresCount);
+  ++numFailures_;
+}
+
 int64_t SharedArbitrator::maxReclaimableCapacity(const MemoryPool& pool) const {
```

Comment:
ditto, can be non-member helper in cpp

Reply:
so are relaimableFreeCacpacity(), reclaimableUsedCapacity(), minGrowCapacity(), etc.